docs/service-catalog/05-04-etcd-stateful.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Etcd-stateful sub-chart
+type: Configuration
+---
+
+To configure the Etcd-stateful sub-chart, override the default values of its `values.yaml` file. This document describes parameters that you can configure.
```

Comment:
```suggestion
To configure the `etcd-stateful` sub-chart, override the default values of its `values.yaml` file. This document describes parameters that you can configure.
```
The formatting of the sub-chart according to what I found here: https://kyma-project.io/docs/1.2/components/service-catalog/#details-service-catalog-backup-and-restore

Reply:
Discussed